lib/views/home_view.dart

```diff
@@ -179,7 +179,7 @@ class _HomeState extends State<Home> {
             text: 'GOOD EVENING,\n',
           ),
           TextSpan(
-            text: widget.user.name.toString(),
+            text: user.displayName,
```

Comment:
does this work for email auth too?

Reply:
Yes, because I am using updateProfile method in getUserDetailsView. See line 209 in getUserDetailsView.